src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -182,6 +182,7 @@ public void run() {
     private final double _minPartitionsToBrokersRatio;
     private final int _minPartitionNum;
     private final TopicFactory _topicFactory;
+    private final Properties _topicProperties;
```

Comment:
Thanks for updating the commit message of the other pull request. Because that forced push has changed the hash of that commit, can you rebase this pull request onto the latest master so that we won't include the changes in `MultiClusterTopicManagementService.java` and `TopicManagementServiceConfig.java` as part of this pull request?